daemon/api_users_test.go

```diff
@@ -63,13 +63,18 @@ func (s *userSuite) SetUpTest(c *check.C) {
 	userLookup = mkUserLookup(s.mockUserHome)
 	s.oldUserAdmin = hasUserAdmin
 	hasUserAdmin = true
+
+	// make sure we don't call these by accident
+	osutilAddUser = nil
```

Comment:
Maybe like so:
```
osutilAddUser = func(name string, opts *osutil.AddUserOpts) { c.Fatalf("unexpected add user %q call", name) }
osutilDelUser = func(name string, opts *osutil.DelUserOpts) { c.Fatalf("unexpected del user %q call", name) }
```

Reply:
I'll do this in #8061, as I'll be pushing some changes there anyway and this one is green :-)